tests/tbasic

```diff
@@ -138,4 +138,13 @@ req -new -batch -key "${PRIURI}" -out ${TMPPDIR}/rsa_csr.pem'
 ossl '
 req -in ${TMPPDIR}/rsa_csr.pem -verify -noout'
 
+title PARA "Test fetching public keys without PIN"
+ORIG_OPENSSL_CONF=${OPENSSL_CONF}
+sed "s/^pkcs11-module-token-pin.*$/##nopin/" ${OPENSSL_CONF} > ${OPENSSL_CONF}.nopin
+OPENSSL_CONF=${OPENSSL_CONF}.nopin
+ossl 'pkey -in $BASEURI -pubin -pubout -out ${TMPPDIR}/rsa.pub.nopin.pem'
+ossl 'pkey -in $ECPUBURI -pubin -pubout -out ${TMPPDIR}/ec.pub.nopin.pem'
```

Comment:
nit: the `BASEURI` and `ECPUBURI` are not the same counterparts. I think the first should be `PUBURI`. The `BASEURI` probably works because of the type is added somehow along the way or the public key is first (and visible without login?)